### ISSUE TITLE
ALE began to change color to white when adding its underscore

### ISSUE THREAD

**Issue by andreyorst** [w0rp/ale#1446]
## Information

**VIM version**
NVIM v0.2.2
Build type: Release

Operating System: Arch LInux, Android (Termux)

## What went wrong
Before update:
![image](https://user-images.githubusercontent.com/19470159/37835296-e39c8760-2ec0-11e8-8c5e-91bef87b4543.png)
After:
![image](https://user-images.githubusercontent.com/19470159/37835364-0def3170-2ec1-11e8-85fd-8d66670786d0.png)
![image](https://user-images.githubusercontent.com/19470159/37834862-d4852670-2ebf-11e8-8eb3-bc8918aa3d68.png)

## Reproducing the bug

Steps for repeating the bug:

1. Write some code with errors.
2. Ugly!

The main reason is as I type in vim ale lints code, and color of letters is binking a lot now.
![image](https://user-images.githubusercontent.com/19470159/37836465-8f11907a-2ec3-11e8-8309-3caac85c7641.png)
![image](https://user-images.githubusercontent.com/19470159/37836481-9c00e0c4-2ec3-11e8-98f1-e8d9c4313dac.png)
![image](https://user-images.githubusercontent.com/19470159/37836509-ad3a3e3a-2ec3-11e8-8bbb-38bfdfc3fdbf.png)
![image](https://user-images.githubusercontent.com/19470159/37836546-c03d8adc-2ec3-11e8-9211-451632aa6c72.png)
And it feels distracting, and somehow laggy.

**Comment by w0rp** [comment]
I can't repeat the bug with those steps. Did you update any other plugins? Are you using a particular colorscheme? Which version of ALE were you running before, and which version are you running now? I can't think of any colors or highlights that have been changed in ALE in quite a while.

**Comment by andreyorst** [comment]
> Which version of ALE were you running before, and which version are you running now?

Please give me a couple of hours, I'll get home to my laptop, where ale might not be updated, and I'll give you more info.

> Did you update any other plugins?

Well, yes. I use vundle.vim, so usually I update the whole list of plugins. You can look through my [list of plugins](https://github.com/andreyorst/dotfiles/blob/master/config/nvim/init.vim) and [ALE settings](https://github.com/andreyorst/dotfiles/blob/master/config/nvim/plugin_configs.vim#L18) if it can help, but I've didn't majorly change anything in a while.
> Are you using a particular colorscheme?

Yes. It is Tomorrow Night form [chriskempson/base16-vim](https://github.com/chriskempson/base16-vim).

**Comment by andreyorst** [comment]
@w0rp, I've checked ALE version at home via `git rev-parse origin/master` and it's stands as ec5750f57b6b98b46cf041f32bcc5d890f3e7e37. I've updated it to dbf530e87fde81397712a85ed87b6a15131afd4c wich is current master. And issue isn't reproduces. So you right, this is not ALE's problem but with another plugin. Sorry for raising invalid issue. I've thought that it is ALE, because only ALE plugin in my config adds such underscore.
